src/components/CommonComponents/TodoIconButtons.js

```diff
@@ -0,0 +1,13 @@
+import React from "react";
+import { IconButton } from "@mui/material";
+import { StyledEngineProvider } from "@mui/styled-engine";
+const TodoIconButtons = ({ onClick, className, value }) => {
+  return (
+    <StyledEngineProvider injectFirst>
+      <IconButton className={className} onClick={() => onClick()} >
+        {value}
+      </IconButton>
+    </StyledEngineProvider>
+  );
+};
+export default TodoIconButtons;
```

Comment:
<img width="349" alt="image" src="https://user-images.githubusercontent.com/17162601/217514458-6392657d-5341-4b9b-aad7-04b5d98b3619.png">
 Fix this EOF issue.